### PR TITLE
improve logging std out

### PR DESCRIPTION
**Pull request by naushniki**
Waiting for jenkins jobs:
https://jenkins.arangodb.biz:3456/view/Upgrade/job/Upgrade-3.8-to-3.9.Linux/481/
https://jenkins.arangodb.biz:3456/view/Upgrade/job/Upgrade-3.7-to-3.8.Windows/102/

### CODE REVIEW

File: release_tester/arangodb/async_client.py [hunk 1]
```diff
@@ -75,9 +75,9 @@ def run_arango_tool_monitored(
         self,
         executeable,
         more_args,
-        timeout,
-        result_line,
-        verbose,
+        timeout=300,
```
Comment (dothebart) [inline release_tester/arangodb/async_client.py:78]: a launched program sitting dead in the water for 5 minutes? I'd lay 1 minute is more than enough.

**Review by dothebart — APPROVED** [review]
please decrease the default timeout for the progressive executor, else LGTM.

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz:3456/view/Upgrade/job/Upgrade-3.8-to-3.9.Linux/482/
https://jenkins.arangodb.biz:3456/view/Upgrade/job/Upgrade-3.7-to-3.8.Windows/103/

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz:3456/view/Upgrade/job/Upgrade-3.8-to-3.9.Linux/491/

**Pull request merged**